### PR TITLE
Add more logging around vote extensions

### PR DESCRIPTION
**Pull request by james-chf**
This PR adds more logging to help with debugging https://github.com/anoma/namada/issues/767 when it occurs in CI in the future.

### CODE REVIEW

**Comment by james-chf** [comment]
pls update wasm

**Comment by james-chf** [comment]
Having issues with the `test_genesis_validators` and `double_signing_gets_slashed` e2e tests in CI, despite these tests passing locally.

e.g. log from a validator in the `double_signing_gets_slashed` test - 
[1668001725376454-namadan-9193308055474754229.log](https://github.com/anoma/namada/files/9971737/1668001725376454-namadan-9193308055474754229.log)

**Review by sug0 — APPROVED** [review]

File: apps/src/lib/cli.rs [hunk 1]
```diff
@@ -2449,8 +2447,8 @@ pub mod args {
             let broadcast_only = BROADCAST_ONLY.parse(matches);
             let ledger_address = LEDGER_ADDRESS_DEFAULT.parse(matches);
             let initialized_account_alias = ALIAS_OPT.parse(matches);
-            let fee_amount = GAS_AMOUNT.parse(matches);
-            let fee_token = GAS_TOKEN.parse(matches);
+            let fee_amount = FEE_AMOUNT.parse(matches);
```
Comment (batconjurer) [inline apps/src/lib/cli.rs:2450]: This will conflict with the bridge pool cli changes. There `Gas` is our gas and `Fee` is the relay fee.

**Review by batconjurer — CHANGES_REQUESTED** [review]

File: apps/src/lib/cli.rs (hunk 1 above)
Comment (james-chf) [reply apps/src/lib/cli.rs:2450]: will update to be like as in https://github.com/anoma/namada/pull/737

**Comment by james-chf** [comment]
pls update wasm

**Comment by james-chf** [comment]
This PR is now just adding logging to help with debugging https://github.com/anoma/namada/issues/767 (which is causing e2e CI to fail)

**Review by batconjurer — APPROVED** [review]

**Pull request merged**